.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,29 @@
+name: aws-deploy
+
+on: push
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.NEXTFLOW_PROD_CI_SECRET_ACCESS_KEY }}
+
+jobs:
+  pre-commit:
+    name: Run pre-commit hooks against all files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-deploy:
+    name: Deploy CloudFormation templates using sceptre
+    runs-on: ubuntu-latest
+    needs: pre-commit
+    if: github.ref == 'refs/heads/prod'
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Deploy 'prod' templates
+        uses: Rurquhart/sceptre-action@v1.0
+        with:
+          sceptre_subcommand: launch --yes dev
```

Comment:
think this should be `launch --yes prod`

Reply:
also it looks like the default is `sceptre_version` is 2.3.0, https://github.com/Rurquhart/sceptre-action/blob/master/action.yml#L14

however the latest version is 2.4.0, https://github.com/Sceptre/sceptre/tags

we should be using the latest version.

Reply:
Thanks to both of you!